tests/vxlan/test_vxlan_decap.py

```diff
@@ -197,6 +199,10 @@ def test_vxlan_decap(setup, vxlan_status, duthosts, rand_one_dut_hostname, tbinf
     logger.info("vxlan_enabled=%s, scenario=%s" % (vxlan_enabled, scenario))
     log_file = "/tmp/vxlan-decap.Vxlan.{}.{}.log".format(
         scenario, datetime.now().strftime('%Y-%m-%d-%H:%M:%S'))
+
+    if skip_traffic_test is True:
+        logger.info("Skip traffic test")
+        return
```

Comment:
Why not use `ptf_runner` to do the skipping?

Reply:
I prefer to use skip_traffic_test fixture because it's easier to read, extend and print log, maybe we will have some platforms need to skip traffic test in the future, but currently ptf_runner skip could only support KVM platform